### PR TITLE
Add SIC codes to the lookupNumber response

### PR DESCRIPTION
**Pull request by kielabokkie**
This PR adds the SIC codes to the response of the `lookupNumber()` request.

The API only returns an array of SIC codes so we use a custom setter to augment the response to add the description of the SIC code.

Closes https://github.com/hyraiq/topcat/issues/8193

### CODE REVIEW

File: src/SicCodes.php [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+    {
+        $codes = self::getCodes();
+
+        return $codes[$code] ?? null;
+    }
+
+    /**
+     * @return array<array-key, string>
+     */
+    public static function getCodes(): array
```
Comment (ndench) [inline src/SicCodes.php:26]: Can you add a comment for where you got this data from?

File: src/Model/CompanyResponse.php [hunk 2]
```diff
@@ -49,6 +50,26 @@ final class CompanyResponse extends AbstractResponse
...
+    public function setSicCodes(array $sicCodes): void
+    {
+        $this->sicCodes = \array_map(
+            fn (string $code): array => [
+                'code'        => $code,
+                'description' => SicCodes::getDescriptionByCode($code),
+            ],
+            $sicCodes
+        );
+    }
```
Comment (ndench) [inline src/Model/CompanyResponse.php:71]: This would be perfect for a custom denormalizer ;)

**Review by ndench — APPROVED** [review]
Just add in a comment as to why the sic codes need be hardcoded and where you got them from.

I custom denormalizer would be cool as well, but largely unnecessary /🤷

**Review by tomtomau — APPROVED** [review]

File: src/Model/CompanyResponse.php (hunk 2 above)
Comment (kielabokkie) [reply src/Model/CompanyResponse.php:71]: Yeah, I thought about it but thought this was easier and less of a hassle to use the library on a project :)

**Pull request merged**